modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+
+Example:
+```bash
+systemctl status redpanda-tuner
+```
+
+Output:
+```bash
+● redpanda-tuner.service - Redpanda Tuner
+     Loaded: loaded (/lib/systemd/system/redpanda-tuner.service; enabled; vendor preset: enabled)
+     Active: active (exited) since Mon 2024-03-25 12:03:51 UTC; 48min ago
+    Process: 3795 ExecStart=/usr/bin/rpk redpanda tune all $CPUSET (code=exited, status=0/SUCCESS)
+   Main PID: 3795 (code=exited, status=0/SUCCESS)
+
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: cpu                    true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_irq               true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_nomerges          true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_scheduler         true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_write_cache       false    true     false      Disk write cache tuner is only supported in GCP
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: fstrim                 false    false    true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: net                    true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: swappiness             true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: transparent_hugepages  false    false    true
+Mar 25 12:03:51 ip-172-31-10-199 systemd[1]: Finished Redpanda Tuner.
+```
+
+Ensure that xref:reference:rpk/rpk-iotune.adoc[`rpk iotune run`] has been run on all hosts. Ensure that the mountpoint listed in this configuration file matches the mountpoint for Redpanda's data directory, usually configured at `/var/lib/redpanda`. See xref:deploy/deployment-option/self-hosted/manual/production/production-deployment.adoc#generate-optimal-io-configuration-settings[Generate optimal I/O configuration settings].
+
+Example:
+```bash
+cat /etc/redpanda/io-config.yaml
+disks:
+  - mountpoint: /mnt/vectorized
+    read_iops: 413115
+    read_bandwidth: 1882494592
+    write_iops: 182408
+    write_bandwidth: 788050688
+```
+
+[[disk-perf]]
+=== Run self test to check disk performance
+
+Run the self test to ensure that disk performance is within an acceptable range.
+
+Example:
+```bash
+rpk cluster self-test status
+```
+
+Output:
+```bash
+NODE ID: 1 | STATUS: IDLE
+=========================
+NAME        512KB sequential r/w throughput disk test
+INFO        write run
+TYPE        disk
+TEST ID     e13b2c93-2417-458b-87be-fac409089513
+TIMEOUTS    0
+DURATION    30000ms
+IOPS        984 req/sec
+THROUGHPUT  492.1MiB/sec
+LATENCY     P50     P90     P99     P999    MAX
+            4095us  4095us  4351us  4607us  5119us
+```
+
+[[hostnames-interfaces]]
+=== Advertised hostnames use correct interfaces
+
+Ensure that the advertised hostnames are operating on the correct network interfaces.
```

Comment:
How would the user know what the correct network interfaces are? Link to topic that provides this guidance? Specify what they are here? We keep saying things like ensure something falls in a range, but then do not specify the range. Or, ensure you are operating on the correct network interfaces, but don't give users guidance on what those are...should the output in all of the examples include the answers/values we tell users to look for? If so, we need to be more clear about what ranges or output values users need.